docker-compose.yml

```diff
@@ -27,6 +27,18 @@ services:
     ports:
       - 3000:3000
 
+  nexus:
+    # Run as the root user so that the container can write to the volume
+    user: "0:0"
```

Comment:
How come that's not needed for the other containers?

Reply:
There's a "Persistent Data" section that might be helpful: https://hub.docker.com/r/sonatype/nexus3/

Also, they mention a Red Hat certified image, have you considered using that?

Reply:
I'll change it to use a named volume.

The image being used is based on UBI 8 so I assumed that was good enough. Did you want one specifically running on RHEL?

Reply:
I think the certified Red Hat image is the same image, just from the Red Hat registry.

Reply:
@lcarva don't you need to login to the Red Hat container registry before using it? If so, I'll just stuck with the one from Docker Hub since this is just for development.

Reply:
You're right. For the sake of usability, let's use the current one.